docs/reference/components.cue

```diff
@@ -240,6 +240,16 @@ components: {
 	#FeaturesGenerate: {
 	}
 
+	#FeaturesSendBufferBytes: {
+		enabled:        bool
+		relevant_when?: string
+	}
+
+	#FeaturesReceiveBufferBytes: {
+		enabled:        bool
+		relevant_when?: string
+	}
```

Comment:
What made you want to add these as features? Adding them as regular old options should be fine. We use features for shared functionality, like batching, but this is specific to sockets, right?

Reply:
Just discussed this on Discord. It's also used in other sinks that can't be shared with the `socket` config, e.g. papertrail.